woocommerce/woocommerce-ios
# Use Fastlane Match for AppStore code signing

This mirrors the changes made for WPiOS in https://github.com/wordpress-mobile/WordPress-iOS/pull/12607.

Currently, we create and manage our certificates and provisioning profiles on an ad-hoc basis. This means that certs and profiles need to be created and renewed manually, but more importantly, they need to be downloaded and installed manually. As well as headaches with installing the right cert, and with renewals, this means there is no way for CI to easily install and use our certs/profiles.

This PR updates our internal (enterprise) and AppStore builds to use [Fastlane Match](https://docs.fastlane.tools/actions/match/) to create and manage their certs/profiles. They are [stored encrypted in Google Cloud](https://docs.fastlane.tools/actions/match/#google-cloud-storage).

As in WPiOS, we may want to extend our usage here to also use this for development builds, but its probably best to do this in small steps.

To test:

- Run `bundle exec fastlane run configure_apply` and then `bundle exec fastlane build_and_upload_itc`.
- It will check that certs/profiles are up to date and install them if needed.
- The AppStore build will fail at the final step because the current version already exists.

Update release notes:
